lumen/ai/views.py

```diff
@@ -40,6 +40,8 @@ class LumenOutput(Viewer):
 
     title = param.String(allow_None=True)
 
+    disabled = param.Boolean(default=False)
```

Comment:
Could start keeping parameters in alphabetical order, unless there's some overriding reason to group them in some other way.